### PR TITLE
C3DC - 955

### PR DESCRIPTION
**Pull request by Senait7-ux**
*Release Notes Page Implementation

### CODE REVIEW

**Comment by Senait7-ux** [comment]
https://github.com/CBIIT/bento-c3dc-frontend/pull/152

File: src/pages/releaseNote/releaseNote.js [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+line-height: 23.2px;
+text-align: left;
+}
+
+
+
+.releaseNoteSection .title{
+   color: #00838F;
+   font-size: 30px;
+   font-weight: 900;
```
Comment (huuaho) [inline src/pages/releaseNote/releaseNote.js:92]: 500

Comment (huuaho) [inline src/pages/releaseNote/releaseNote.js:92]: ~~font-weight: 900;~~
font-weight: 500;

File: src/pages/releaseNote/releaseNote.js [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+ }
+ .version:nth-child(even){
+   background-color: #DBE7E980;
+ }
+.descriptionDate {
+      font-family: Inter;
+      font-size: 13px;
+      font-weight: 300;
+      line-height: 24px;
+      text-align: left;
```
Comment (huuaho) [inline src/pages/releaseNote/releaseNote.js:230]: text-transform: Uppercase;

File: src/pages/releaseNote/releaseNote.js [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+line-height: 23.2px;
+text-align: left;
+
+}
+.second-title {
+font-family: Nunito;
+font-size: 16px;
+font-weight: 700;
+line-height: 23.2px;
+text-align: left;
```
Comment (huuaho) [inline src/pages/releaseNote/releaseNote.js:84]: color: #00000

File: src/pages/releaseNote/releaseNote.js [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+   display: flex;
+   flex-direction: column;
+   padding: 25px;
+   padding-top: 15px;
+   overflow-y: scroll;
+font-family: Nunito;
+font-size: 16px;
+font-weight: 400;
+line-height: 23.2px;
+text-align: left;
```
Comment (huuaho) [inline src/pages/releaseNote/releaseNote.js:76]: color: #000000

File: src/bento/dashboardTabData.js [hunk 5]
```diff
@@ -1597,7 +1597,7 @@ export const tabContainers = [
       {
         dataField: 'toronto_childhood_cancer_staging',
         header: 'Toronto Childhood Cancer Staging',
-        display: false,
+        display: true,
```
Comment (huuaho) [inline src/bento/dashboardTabData.js:1600]: Where is it said that we need this to be visible on this ticket? Was this change intentional?

File: src/pages/releaseNote/releaseNote.js [hunk 6]
```diff
@@ -0,0 +1,368 @@
...
+`;
+
+
+
+function ReleaseNotePage() {
+  const [list_versions, setListVersions] = useState(releaseNotePageData);
+  const [selectedVersion, setSelectedVersion] = useState(defaultVersion);
+
+
+  const handelYearSelection = index => {
```
Comment (huuaho) [inline src/pages/releaseNote/releaseNote.js:266]: ~~handle~~ 
handle

File: src/pages/releaseNote/releaseNote.js [hunk 7]
```diff
@@ -0,0 +1,368 @@
...
+    });
+
+
+    setListVersions(versions);
+  };
+
+
+
+
+  const handelCheckbox = (yearIndex, versionIndex, version) => {
```
Comment (huuaho) [inline src/pages/releaseNote/releaseNote.js:281]: ~~handle~~ 
handle

File: src/pages/releaseNote/releaseNote.js [hunk 8]
```diff
@@ -0,0 +1,368 @@
...
+      <div className='releaseNoteBody'>
+        <div>
+          <div className='versionSelector'>
+            <div className='yearsectionTitle'>{'RELEASE NOTES'}</div>
+            {list_versions.map((release, yearIndex) => {
+              return (
+                <>
+                  <button
+                    onClick={() => {
+                      handelYearSelection(yearIndex);
```
Comment (huuaho) [inline src/pages/releaseNote/releaseNote.js:324]: ~~handle~~ 
handle

File: src/pages/releaseNote/releaseNote.js [hunk 9]
```diff
@@ -0,0 +1,368 @@
...
+                  </button>
+                  <div className='version_list'>
+                    {release.is_open &&
+                      release.versions.map((versions, versionIndex) => {
+                        return (
+                          <button className='version'>
+                            <div>
+                              <input
+                                onChange={() => {
+                                  handelCheckbox(yearIndex, versionIndex, versions.Version);
```
Comment (huuaho) [inline src/pages/releaseNote/releaseNote.js:339]: ~~handle~~ 
handle

File: src/pages/releaseNote/releaseNote.js [hunk 10]
```diff
@@ -0,0 +1,368 @@
...
+   height: 800px;
+ }
+.versionSelector {
+   width: 303px;
+   height: 314px;
+   border: 2px #4D7874 solid;
+   border-radius: 20px;
+   background-color: white;  
+   overflow:hidden;
+   overflow-y: scroll;
```
Comment (huuaho) [inline src/pages/releaseNote/releaseNote.js:59]: ~~overflow-y: scroll;~~
overflow-y: auto;

**Comment by huuaho** [comment]
Great work Seni! It's awesome see stuff come up from scratch, however there are a few things I would like to see a bit more polished to match the Figma design.

I've written comments for quick fixes for some discrepancies that can be quickly fixed but there are some things I believe that require a little more attention.

The first thing that stuck out to me was the scroll bars. They are square and break the flow of the curved edges, they should be stylized to match the ones found in Figma. For the version selector, we can do a quick fix of setting it to auto, but in the future when the scroll bar would appear, the styling would still be mismatched. It's best to see if we can fix it now.

There's a detail that needs clarifying - there's a little bit of a mismatch with the dates. In the release notes we have the full dates (all uppercase). But in the version selector we have the shorthand of the dates (January -> JAN, July -> JUL, March -> MAR, etc). I'm not sure if this is a deal breaker with the PM team. We should ask if this this matters in the spec.

The spacing of the fields (ul element) do not match the spacing according to the design. We should reduce the margin on them to fit the design.

Again, amazing Seni. I hope we can make it even better.

File: src/pages/releaseNote/releaseNote.js (hunk 10 above)
Comment (Senait7-ux) [reply src/pages/releaseNote/releaseNote.js:59]: Any reason for this update request? scroll handled the desired outcome

File: src/bento/dashboardTabData.js (hunk 5 above)
Comment (Senait7-ux) [reply src/bento/dashboardTabData.js:1600]: This was merged , we had a lot of conflicts which I resolved a while back, and this change is from the conflict merge

Comment (Senait7-ux) [reply src/bento/dashboardTabData.js]: resolved

File: src/pages/releaseNote/releaseNote.js (hunk 4 above)
Comment (Senait7-ux) [reply src/pages/releaseNote/releaseNote.js]: resolved

File: src/pages/releaseNote/releaseNote.js (hunk 2 above)
Comment (Senait7-ux) [reply src/pages/releaseNote/releaseNote.js:246]: resolved

File: src/pages/releaseNote/releaseNote.js (hunk 1 above)
Comment (Senait7-ux) [reply src/pages/releaseNote/releaseNote.js]: resolved

File: src/pages/releaseNote/releaseNote.js (hunk 6 above)
Comment (Senait7-ux) [reply src/pages/releaseNote/releaseNote.js]: resolved

File: src/pages/releaseNote/releaseNote.js (hunk 7 above)
Comment (Senait7-ux) [reply src/pages/releaseNote/releaseNote.js]: resolved

File: src/pages/releaseNote/releaseNote.js (hunk 8 above)
Comment (Senait7-ux) [reply src/pages/releaseNote/releaseNote.js]: resolved

File: src/pages/releaseNote/releaseNote.js (hunk 9 above)
Comment (Senait7-ux) [reply src/pages/releaseNote/releaseNote.js]: resolved

File: src/pages/releaseNote/releaseNote.js (hunk 10 above)
Comment (Senait7-ux) [reply src/pages/releaseNote/releaseNote.js:65]: resolved

**Review by iksheth — APPROVED** [review]
Approved PR. Additional changes will be coming on the way as User story is updated. Merging for now.
